modules/transport.js

```diff
@@ -525,6 +528,9 @@ Transport.prototype.shared = {
 	 * @todo Add description of the function
 	 */
 	postBlock(query) {
+		if (!library.config.broadcasts.on) {
```

Comment:
Do we need these checks here? My understanding is that these functions will never be triggered if 'broadcasterNextRelease' job is not registered.